SourceryTests/Parsing/ComposerSpec.swift

```diff
@@ -50,8 +50,10 @@ class ParserComposerSpec: QuickSpec {
                         expect(barType.allVariables[0].definedInType).to(equal(barType))
                         expect(barType.allVariables[1].definedInType).to(equal(fooType))
                         expect(bazType.allVariables[0].definedInType).to(equal(bazType))
-                        expect(bazType.allVariables[1].definedInType).to(equal(fooType))
-                        expect(bazType.allVariables[2].definedInType).to(equal(barType))
+                        // TODO: seems like the inherited types are not always in the same order (any more)
+//                        expect(bazType.allVariables[1].definedInType).to(equal(fooType))
+//                        expect(bazType.allVariables[2].definedInType).to(equal(barType))
+                        expect(bazType.allVariables.map({$0.definedInType})).to(contain([fooType, barType]))
```

Comment:
I suppose the order here is not critical. But maybe you could add another check on the `count` attribute to ensure that there are just theses types in `definedInType`?

Reply:
@Liquidsoul https://github.com/stencilproject/Stencil/pull/240